### ISSUE TITLE
Request: Show list of permissions

### ISSUE THREAD

**Issue by kwlg** [M66B/XPrivacy#862]
This is a request for lazy people like myself and therefore not an important one, but I would find it useful to see the list of permissions an app has where the list is in the same format as in the Play Store or in the System Settings - Apps - <App's name>. For example,

"Your location
approximate location (network-based)
Your messages
receive text messages (SMS)
read your text messages (SMS or MMS)
Network communication
full network access"

Currently, there are green keys for functions that the app has access to. This is good as these show what the app can in fact do. However, many apps can access e.g. certain location related functions without having the right permissions declared in the manifest. Deciding whether or not to restrict them then depends on the actual permissions the app has declared. Of course, one could simply press Menu - Application - App details in XPrivacy to see the list of permissions, but according to me, it would be easier to make restrictions for apps if the permissions were shown directly in XPrivacy.

In summary: The green keys are good and should be kept the way they are, but it would be easier to make the restrictions for an (inexperienced) user if the list of the permissions were also shown. The green keys in combination with such a list would be very useful in my opinion.
